src/Apps/Artist/Routes/AuctionResults/ArtistAuctionDetailsModal.tsx

```diff
@@ -17,7 +17,7 @@ import { get } from "Utils/get"
 
 interface Props {
   hideDetailsModal?: () => void
-  auctionResult: {
+  auctionResult?: {
```

Comment:
Would one ever want to trigger the modal without a result?

Reply:
This modal will go away in the new design. I just wanted to get rid of `ts` errors for now.